agenta-backend/agenta_backend/services/evaluators_service.py

```diff
@@ -29,14 +32,40 @@ def auto_similarity_match(
     return result
 
 
-def auto_regex_test(_, test_string: str, settings_values: dict) -> Result:
+def auto_regex_test(
+    variant_output: str, correct_answer: str, settings_values: Dict[str, Any]
+) -> Result:
     re_pattern = re.compile(settings_values["regex_pattern"], re.IGNORECASE)
     result = (
-        bool(re_pattern.search(test_string)) == settings_values["regex_should_match"]
+        bool(re_pattern.search(variant_output)) == settings_values["regex_should_match"]
     )
     return Result(type="bool", value=result)
 
 
+def auto_webhook_test(
```

Comment:
please remove network logic from this layer and move it to the router layer.
This method should only handle the evaluation